src/main/java/com/aspire/student/controller/StudentController.java

```diff
@@ -148,7 +148,9 @@ public void saveStudent(ActionRequest actionRequest, ActionResponse actionRespon
     try {
       student = studentService.saveStudent(student);
     } catch (Exception e) {
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1166)